src/api/tests/service/profile/test_profile.py

```
@@ -0,0 +1,99 @@
+        app=None, parent_id="shifu_bid", user_id="user_bid"
+    )
+
+    assert result == ["updated"]
+    assert ("unused", "shifu_bid") in calls
+    assert ("update", "shifu_bid", ("v1", "v2"), True, "user_bid") in calls
+
+
+def test_get_profile_variable_usage_groups_keys(monkeypatch):
+    calls = []
+
+    def fake_get_defs(_app, parent_id=None, type="all"):
```

Ruff (0.14.13)

Unused function argument: `type`

(ARG001)